File: react/reactproject/src/models/Message.ts
export class Message {
    MessageId: number;
    UserId: number;
    ReceiverId: number;
    MessageContent: string;
    MessageDate: Date;
    constructor(data: Partial<Message> = {}) {
        this.MessageId = data.MessageId ?? 0;
        this.UserId = data.UserId ?? 0;
        this.ReceiverId = data.ReceiverId ?? 0;
        this.MessageContent = data.MessageContent ?? "defalt content";
        this.MessageDate = data.MessageDate ?? new Date();
    }
}

File: Net/FinalProject/FinalProject.Clean/Models/Message.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FinalProject.Core.Models
{
    public class Message
    {
        [Key]
        public int MessageId { get; set; }
        public int UserId { get; set; }
        public int ReceiverId { get; set; }
        public string MessageContent { get; set; }
        public DateTime MessageDate { get; set; }
        public User User { get; set; }
        public Doctor Doctor { get; set; }
    }
}
